src/components/groups/AddPlayerDialog.tsx

```diff
@@ -85,14 +90,28 @@ const AddPlayerDialog = (props: Props) => {
     } else {
       form.reset();
       onPlayerAdd();
-      setOpen(false);
+      onClose();
     }
   };
 
+  const onOpenChange = (open: boolean) => {
+    setFreshVisit(false);
+    setOpen(open);
+  }
+
+  const onClose = () => {
+    onOpenChange(false);
+  }
+
   return (
-    <Dialog open={open} onOpenChange={setOpen}>
+    <Dialog open={open || freshVisit} onOpenChange={onOpenChange}>
       <DialogTrigger asChild>
-        <Button>Add Player</Button>
+        <Button variant='outline'>
+          <div className='flex gap-1 items-center'>
+            <FontAwesomeIcon icon={faUser} />
```

Comment:
I don't think it's obvious that this is the button that you press to add a user.

<img width="489" alt="Screenshot 2023-08-26 at 1 18 38 PM" src="https://github.com/pohang/baddybuddy/assets/19980697/50bc8776-f30d-46c2-af2d-6857fdca7de4">

Reply:
maybe toss a plus sign to the end or something?

Reply:
updated icon to user with plus sign